mq/mq.lua

```diff
@@ -267,6 +265,25 @@ function TLO.Heading(y, x) end
 ---@return heading
 function TLO.Heading(north, west) end
 
+---Return the item specified by inventory slot name
+---@param name string
+---@return invslot
+function TLO.InvSlot(name) end
```

Comment:
I think this one was supposed to be Inventory?

Reply:
Must be something else given the Inventory datatype is not indexable

Reply:
I was thinking TLO.Inventory, dissregard my comment